src/main/scala/org/broadinstitute/dsde/workbench/leonardo/config/SwaggerConfig.scala

```diff
@@ -1,3 +1,5 @@
 package org.broadinstitute.dsde.workbench.leonardo.config
 
-case class SwaggerConfig(googleClientId: String, realm: String)
+import org.broadinstitute.dsde.workbench.leonardo.model.GoogleClientId
+
+case class SwaggerConfig(googleClientId: GoogleClientId, realm: String)
```

Comment:
We're still stringly-typing confs, right?

Reply:
Yeah I suppose we are. I can make this `String` to be consistent.